tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1788,6 +1788,10 @@ class TorchLlmArgs(BaseLlmArgs):
     moe_backend: str = Field(default='CUTLASS',
                              description="MoE backend to use.")
 
+    moe_disable_finalize_fusion: bool = Field(
+        default=False,
+        description="Disable FC2+finalize kernel fusion in CUTLASS MoE backend")
```

Comment:
Can you clarify why someone would want to do this, there's no documentation anywhere I can see that tells someone that this will resolve non-determinism